### DISCUSSION TITLE
How to deploy application using HELM

### DISCUSSION THREAD

**Discussion by tusharau** [gruntwork-io/knowledge-base#663] [How do I]
Hello Team,

We want to deploy our application using custom HELM configuration, can you please let us know which module we should refer for the deployment process.

Thanks
Tushar Shukla

---

<ins datetime="2023-02-09T07:45:07Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109886">Tracked in ticket #109886</a></p>
</ins>

**Comment by gitsstewart** [comment]
@pras111gw recommends the following:

After deploying EKS using the terraform-aws-eks (https://github.com/gruntwork-io/terraform-aws-eks) repo, have a look at our samples in helm-kubernetes-services (https://github.com/gruntwork-io/helm-kubernetes-services). 
Couple of quickstart samples are in that repo https://github.com/gruntwork-io/helm-kubernetes-services/tree/main/examples
